Wsl.Tests.ps1

```diff
@@ -230,6 +230,12 @@ Describe "WslManagementPS" {
         Test-Distro ($distros | Where-Object { $_.Name -eq "wslps_raw" }) "wslps_raw" 2 "Stopped"
     }
 
+    It "Can list online distributions" {
+        # Tests that there are online distributions available
+        $distros = Get-WslDistributionOnline
+        $distros.Count | Should -BeGreaterThan 0

```

Comment:
Thanks for adding a test! This can be simplified a bit:


```suggestion
        $distros | Should -Not -BeNullOrEmpty
```

I think it would also be a good idea to make sure the list does not contain any items with an empty Name or FriendlyName. This should guard against a premature match on the NAME string in the localized text at the top of the command.

Reply:
I've implemented that update, but not done a lot of Pester testing before, so will work out the required checks here.

Reply:
Added a pester Test for this now.